extension/src/project-explorer/ivy-project-explorer.ts

```diff
@@ -35,11 +35,12 @@ export class IvyProjectExplorer {
     registerCmd(`${VIEW_ID}.buildProject`, (selection: TreeSelection) => this.execute('engine.buildProject', selection));
     registerCmd(`${VIEW_ID}.deployProject`, (selection: TreeSelection) => this.execute('engine.deployProject', selection));
     registerCmd(`${VIEW_ID}.buildAndDeployProject`, (selection: TreeSelection) => this.execute('engine.buildAndDeployProject', selection));
-    registerCmd(`${VIEW_ID}.addBusinessProcess`, (selection: TreeSelection) => this.addProcess('Business Process', selection));
-    registerCmd(`${VIEW_ID}.addCallableSubProcess`, (selection: TreeSelection) => this.addProcess('Callable Sub Process', selection));
-    registerCmd(`${VIEW_ID}.addWebServiceProcess`, (selection: TreeSelection) => this.addProcess('Web Service Process', selection));
+    registerCmd(`${VIEW_ID}.addProcess`, (selection: TreeSelection, pid: string) => this.addProcess(selection, undefined, pid));
```

Comment:
will this now also appear on the vscode command palette? because it would be nice IMO if this is a hidden cmd. because it is only triggered by the code

Reply:
no, as the command is not listed in package.json under contributions.